exec/java-exec/src/main/java/org/apache/drill/metastore/ColumnStatistics.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.metastore;
+
+import java.util.Comparator;
+
+/**
+ * Represents collection of statistics values for specific column.
+ *
+ * @param <T> type of column values
+ */
+public interface ColumnStatistics<T> {
```

Comment:
Suggest naming this as ExactColumnStatistics and related implementors of this interface accordingly.  The reason is that we want to avoid confusion with **estimated** column statistics. 

cc @gparai